### ISSUE TITLE
Wave effect is not working well on submit button

### ISSUE THREAD

**Issue by Mr-Zer0** [Dogfalo/materialize#2190]
Wave effect is nice and work well in every button except from form submit buttons. Could you please check it.

**Comment by SuperDJ** [comment]
It isn't very clear what you mean. But it might be because it's a submit button. The reason being is that the wave effect isn't 100% done when the page is submited. Could this be the case?

**Comment by Mr-Zer0** [comment]
<img width="215" alt="screen shot 2015-10-26 at 11 52 54 am" src="https://cloud.githubusercontent.com/assets/2142972/10721842/2506c3b2-7bd8-11e5-9740-6bc26e5a68dd.png">
When I use wave effect on submit button it shows like the above image.

```html

<input type="submit" value="Create Department" class="waves-effect waves-light btn">

```

**Comment by SuperDJ** [comment]
That looks oke. Do you have a other stylesheet in your website that overwrites the material css. What browser are you using?

**Comment by Mr-Zer0** [comment]
I use chrome. I will recheck it. But I have no other css file except from materialize.min.css.

**Comment by acburst** [comment]
Fixed in 02cf1dc
